src/lib/swapper/swappers/ThorchainSwapper/utils/longTailHelpers.ts

```diff
@@ -185,3 +185,16 @@ export const selectBestRate = (
     [undefined, undefined],
   )
 }
+
+export const poolToShortenedPool = {
+  'THOR.RUNE': 'r',
```

Comment:
@NeOMakinG THOR.RUNE isn't a pool, see https://thornode.ninerealms.com/thorchain/pools

Reply:
Actually the previous naming was perfect! We should just remove `THOR.RUNE` as selling from RUNE isn't possible for long-tails so that's really never going to be present in long-tail memos